docs/guides/update-guide/100-to-110.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 100-to-110
+title: Update 1.0 to 1.1
+---
+
+The following sections will explain for each component of the system which
+adjustments have to be made to migrate from Camunda Cloud 1.0.x to 1.1.0
+
+
+## Server
+
+### Operate
+
+With Operate 1.1 a new feature was introduced to navigate between call activity
+hierarchies. This feature is only available for instances started after the
+version update. Older instances will not expose this information to the user.
```

Comment:
I always specify the exact version so no one has to read very far for the right context 😄 

```suggestion
version 1.1 update. Older instances will not expose this information to the user.
```